content/en/hosting/monitoring/setup.md

```diff
@@ -252,43 +252,30 @@ Slack alerts can be configured within the Grafana web GUI for the specific rules
 
 #### Environment Variables
 
-All the variables in the `.env` file:
-
-| Name                        | Default                         | Description                                                                                            |
-|-----------------------------|---------------------------------|--------------------------------------------------------------------------------------------------------|
-| `GRAFANA_ADMIN_USER`        | `medic`                         | Username for the Grafana admin user                                                                    |
-| `GRAFANA_ADMIN_PASSWORD`    | `password`                      | Password for the Grafana admin user                                                                    |
-| `GRAFANA_VERSION`           | `latest`                        | Version of the `grafana/grafana-oss` image                                                             |
-| `GRAFANA_PORT`              | `3000`                          | Port on the host where Grafana will be available                                                       |
-| `GRAFANA_BIND`              | `127.0.0.1`                     | Interface Grafana will bind to.  Change to `0.0.0.0` if you want to expose to all interfaces.  |
-| `GRAFANA_DATA`              | `./grafana/data`                | The host directory where Grafana data will be stored                                                   |
-| `GRAFANA_PLUGINS`           | `grafana-discourse-datasource`  | Comma separated list of plugins to install (e.g: `grafana-clock-panel,grafana-simple-json-datasource`) |
-| `JSON_EXPORTER_VERSION`     | `latest`                        | Version of the `prometheuscommunity/json-exporter` image                                               |
-| `PROMETHEUS_VERSION`        | `latest`                        | Version of the `prom/prometheus` image                                                                 |
-| `PROMETHEUS_DATA`           | `./prometheus/data`             | The host directory where Prometheus data will be stored                                                |
-| `PROMETHEUS_RETENTION_TIME` | `60d`                           | Length of time that Prometheus will store data (e.g. `15d`, `6m`, `1y`)                                |
-
+See the [example .env file](https://github.com/medic/cht-watchdog/blob/main/.env.example) for a list of all environment variables that can be set.
```

Comment:
@mrjones-plip can fight me over this, but in the spirit of not duplicating things (especially when they are so likely to get out of sync) it seems sufficient to just link to the example.